client/src/pages/EditProfilePage/EditProfilePage.tsx

```diff
@@ -78,71 +227,267 @@ const EditProfilePage = () => {
             <input
-              className="bg-gray-800 p-2 rounded text-white w-full"
-              id="email"
-              name="email"
-              type="email"
+              className='bg-gray-800 p-2 rounded text-white w-full'
+              id='email'
+              name='email'
+              type='email'
               value={formData.email}
               onChange={handleChange}
             />
           </label>
-          <label className="block font-bold mb-2 text-sm" htmlFor="personalBio">
-            Personal Bio
+          <label className='block text-sm font-bold mb-2' htmlFor='nickName'>
+            Nickname
             <input
-              className="bg-gray-800 p-2 rounded text-white w-full"
-              id="personalBio"
-              name="personalBio"
-              type="text"
+              className='bg-gray-800 p-2 rounded text-white w-full'
+              id='nickName'
+              name='nickName'
+              type='text'
+              value={formData.nickName}
+              onChange={handleChange}
+            />
+          </label>
+          <label className='block font-bold mb-2 text-sm' htmlFor='cohort'>
+            Cohort
+            <input
+              className='bg-gray-800 p-2 rounded text-white w-full'
+              id='cohort'
+              name='cohort'
+              type='text'
+              value={formData.cohort}
+              onChange={handleChange}
+            />
+          </label>
+          <label className='block font-bold mb-2 text-sm' htmlFor='personalBio'>
+            Personal Bio
+            <textarea
+              className='bg-gray-800 p-2 rounded text-white w-full'
+              id='personalBio'
+              name='personalBio'
               value={formData.personalBio}
               onChange={handleChange}
+              style={{ maxHeight: "200px" }}
+              maxLength={1000}
             />
           </label>
+          <label className='block font-bold mb-2 text-sm' htmlFor='company'>
+            Current Company
+            <input
+              className='bg-gray-800 p-2 rounded text-white w-full'
+              id='company'
+              name='company'
+              type='text'
+              value={formData.careerInformation.currentPosition.company}
+              onChange={handleCompanyChange}
+            />
+          </label>
+          <label className='block font-bold mb-2 text-sm' htmlFor='title'>
+            Current Title
+            <input
+              className='bg-gray-800 p-2 rounded text-white w-full'
+              id='title'
+              name='title'
+              type='text'
+              value={formData.careerInformation.currentPosition.title}
+              onChange={handleTitleChange}
+            />
+          </label>
+
+          <label
+            className='block font-bold mb-2 text-sm'
+            htmlFor='availabilityForNetworking'
+          >
+            Available for Networking?
+            <div
```

Comment:
This div works but for accessibility reasons we might want to use an input checkbox. I might be wrong but I don't think a screen reader would register this as clickable

Reply:
Good call! Instead of using an input checkbox, I added the role of 'button' to this div and set its tabIndex to 0, making it focusable. I believe this should now register as clickable for a screen reader.